environs/config/config.go

```diff
@@ -1839,11 +1877,26 @@ global or per instance security groups.`,
 		Type:        environschema.Tattrs,
 		Group:       environschema.EnvironGroup,
 	},
-	"rsyslog-ca-cert": {
+	RsyslogURL: {
```

Comment:
it it worth noting, somewhere user-readable, that you have to set all the rsyslog-* values together?